component.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright Abridged, Inc. 2022. All Rights Reserved.
```

Comment:
Hmm, please move all files to `src` directory.

Reply:
No problem. All moved over